openpype/hosts/flame/hooks/pre_flame_setup.py

```diff
@@ -19,7 +19,10 @@ class FlamePrelaunch(PreLaunchHook):
     app_groups = ["flame"]
 
     # todo: replace version number with avalon launch app version
-    flame_python_exe = "/opt/Autodesk/python/2021/bin/python2.7"
+    flame_python_exe = (
+        "/opt/Autodesk/python/{OPENPYPE_FLAME_VERSION}"
+        "/bin/python2.7"
+    )
```

Comment:
Not an issue for this PR and current state but it would be better to not require env containing flame version by default. At least require full path to flame's python and if the env is not set or available then try to find python based on executable path. When executable is real path to flame it should be possible...

Reply:
good point, I will convert this to abs path in env var. 
But executable is not the same so the fallback cannot be done this way.

Reply:
resolved by gitkraken://repolink/33c731ef76967f4bd3f6ba8b287ecc0c52412f33/commit/75b828022103a7385a5c25116d1bcb54967fa833?url=https%3A%2F%2Fgithub.com%2Fpypeclub%2FOpenPype.git